README.md

```diff
@@ -264,7 +264,7 @@ or creating the directories and extracting the archive through the File Explorer
 
 The default way to use the libraries provided by vcpkg in CMake is to use the [vcpkg CMake toolchain](https://github.com/Microsoft/vcpkg/blob/master/docs/users/integration.md#cmake-toolchain-file-recommended-for-open-source-cmake-projects), so to use the provided vcpkg workspace you need to pass the `-DCMAKE_TOOLCHAIN_FILE=C:\robotology\vcpkg\scripts\buildsystems\vcpkg.cmake` CMake option when configuring the robotology-superbuild, either from the command line or via the CMake GUI.
 
-Furthermore, as during the build process some programs that use `vcpkg`-install dlls are executed, you need to make sure that the `vcpkg` dll are in the path, by appending to the `PATH` the `C:/robotology/vcpkg/installed/x64-linux/bin` and `C:/robotology/vcpkg/installed/x64-linux/bin` directories.
+Furthermore, as during the build process some programs that use `vcpkg`-install dlls are executed, you need to make sure that the `vcpkg` dll are in the path, by appending to the `PATH` the `C:/robotology/vcpkg/installed/x64-windows/bin`  directory.
```

Comment:
Sorry, the actual directory to add are one more to ensure that everything works  correctly also when compiling in Debug: 
```suggestion
Furthermore, as during the build process some programs that use `vcpkg`-install dlls are executed, you need to make sure that the `vcpkg` dll are in the path, by appending to the `PATH` the `C:/robotology/vcpkg/installed/x64-windows/bin` and `C:/robotology/vcpkg/installed/x64-windows/debug/bin` directories.
```
See https://github.com/robotology/robotology-superbuild/blob/master/packaging/windows/scripts/addPathsToUserEnvVariables-vcpkg.ps1#L33 .